google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -684,6 +707,18 @@ Builder setClock(ApiClock clock) {
       return this;
     }
 
+    /** Gives the ability to enable Open Telemetry Tracing */
```

Comment:
Can we add the language that spans/attributes are subject to change to here and the publisher? Language in python: https://github.com/googleapis/python-pubsub/blob/747a016c6d9936adc0db36584537e6879df5eb93/google/cloud/pubsub_v1/types.py#L204

Reply:
Done